workflow/Activity/DataPipeline Activity.go

```diff
@@ -0,0 +1 @@
+package activity
```

Comment:
remove this file

Reply:
have added the activities related to datapipeline